<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projeto e Analise de Algoritmos (Ciência da Compuação)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="cbOrigem">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInicial">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select initial node:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFinal">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select final node:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbFinal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="MostrarCaminhoButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Caminho</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cdMetodo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DFS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BFS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ordenação Topologica</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dijkstra</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prim</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kruskal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ford-Fulkerson</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelInicial_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Metodo:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGrafos">
    <property name="title">
     <string>Grafos</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuGrafos"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
